pkg/langruntime/langruntime.go

```diff
@@ -240,6 +240,8 @@ func (l *Langruntimes) GetBuildContainer(runtime, depsChecksum string, env []v1.
 	case strings.Contains(runtime, "java"):
 		command = appendToCommand(command,
 			"mv /kubeless/pom.xml /kubeless/function-pom.xml")
+	case strings.Contains(runtime, "ballerina"):
+		command = appendToCommand(command, "ls /kubeless/ballerina.conf")
```

Comment:
I would throw an error here to make it clear that `ballerina` doesn't require this file. something like:

```
+		command = appendToCommand(command, "echo 'ballerina does not require a dependencies file'", "exit 1")
```